src/ddmd/globals.d

```diff
@@ -172,6 +172,10 @@ struct Param
     const(char)* moduleDepsFile;        // filename for deps output
     OutBuffer* moduleDeps;              // contents to be written to deps file
 
+    const(char)* makeDepsFile;          // filename for make-style dependencies output
+    const(char)* makeDepsTarget;        // the generated make target name
+    Array!(const(char)*)* makeDeps;     // array of paths that the compiled module(s) depend on
```

Comment:
The `makeDeps` field could be private, but the `File` and `Target` can be useful for other drivers to input their commandline options to dmdfe. I've basically followed what changes the `-deps` option that was already built into dmd introduced, so that's why I added these fields here.